maven-java/kalix-maven-archetype-spring-value-entity/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,88 @@
+# Quickstart project: Customer Registry with Views
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project demonstrates the use of Value Entity and View components.
+To understand more about these components, see [Developing services]https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation.
+
+
+## Building
+
+Use Maven to build your project:
+
+```shell
+mvn compile
+```
+
+
+## Running Locally
+
+To run the example locally, you must run the Kalix proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Kalix proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+```shell
+docker-compose up
+```
+
+To start the application locally, the `exec-maven-plugin` is used. Use the following command:
+
+```shell
+mvn compile exec:exec
+```
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. 
+
+FIXME update for spring
+
+* Create a customer with:
```

Comment:
Our archetypes use a simple Counter model, not Customer registry. 

it should only contain the Value Entity, not Views. The goal is to make is a simple as possible.

Reply:
@ennru / @beritou , this have been discussed in the past, but maybe worth to revisit it.

If I remember correctly, the reason why we added a counter protobuf in our gRPC archetype is that we have to teach people about protobuf, how to get started with our annotations and have some structure to start from. 

For the SpringSDK, it may make more sense to not have a model at all because the target audience knows already how to code in Java. Once we have the documentation in place, users can start from an empty archetype and code their model in Java right away. 

WDYT?